docs/tutorial/index.md

```diff
@@ -3,37 +3,37 @@ title: Gatsby.js Tutorials
-8.  [Preparing a site to go live](/tutorial/part-eight/): Learn how to audit your site for performance and best practices for accessibility, SEO, and more.
+4.  [Query dữ liệu trong blog](/tutorial/part-four/): Tạo blog và sử dụng QraphQL query để kéo tiêu đề trang web của bạn vào tiêu đề blog.
+5.  [Source plugins và hiển thị dữ liệu đươc truy  vấn](/tutorial/part-five/): Sử dụng source plugin để kéo những bài blog Markdown vào trang web của bạn và tạo một trang mục lục với danh sách tất cả những bài đăng trên blog.
+6.  [Transformer plugins](/tutorial/part-six/): Sử dụng một transformer plugin để chuyển đổi các bài blog Markdown thành một dạng mà blog có thể kết xuất.
+7.  [Tạo trang từ dữ liệu một cách lập trình](/tutorial/part-seven/): Tìm hiểu cách tạo một tập hợp các trang từ những bài đăng trên blog một cách lập trình.
+8.  [Chuẩn bị một trang web để phát hành trực tiếp](/tutorial/part-eight/): Tìm hiểu cách kiểm tra trang web của bạn về hiệu suất và các thực tiễn tốt nhất về khả năng truy cập, SEO và hơn thế nữa.
 
-## Theme tutorials
+## Hướng dẫn về theme
 
-Gatsby themes abstract all of your default configuration (shared functionality, data sourcing, design) out of your site and into an installable package.
+Gatsby themes tóm tắt tất cả cấu hình mặc định của bạn (chức năng cùng chia sẻ, nguồn dữ liệu, thiết kế) ra khỏi trang web của bạn và làm nó thành một package có thể cài đặt.
 
-This means that the configuration and functionality isn’t directly written into your project, but rather versioned, centrally managed, and installed as a dependency. You can seamlessly update a theme, compose themes together, and even swap out one compatible theme for another.
+Điều này có nghĩa là cấu hình và chức năng không được ghi trực tiếp vào project của bạn, nhưng được phiên bản, quản lý tập trung và được cài đặt như một dependency. Bạn có thể cập nhật một theme liền mạch, kết hợp các themes với nhau và thậm chí trao đổi một theme tương thích cho một theme khác.
 
-Learn more about using and building themes with the [Gatsby theme tutorials](/tutorial/theme-tutorials/).
+Tìm hiểu thêm về sử dụng và xây dựng các themes với [Hướng dẫn Gatsby theme](/tutorial/theme-tutorials/).
 
-## Additional tutorials
+## Hướng dẫn bổ sung
 
-There are many use cases for Gatsby, some of which aren't covered in the first set of tutorials to keep you focused while learning. Gatsby's Additional Tutorials section is a collection of resources showing how to complete even more Gatsby tasks step-by-step, such as using source plugins for images and CMS content.
+Có rất nhiều trường hợp sử dụng cho Gatsby, một số không dược đề cập trong phần hướng dẫn đầu tiên để giúp bạn tập trung khi tìm hiểu. Phần hướng dẫn bổ sung của Gatsby là một tập hợp các resource hướng dẫn cách hoàn thành nhiều hơn nữa những bài làm Gatsby theo từng bước, như là sử dụng source plugins cho hình ảnh and nội dung CMS.
```

Comment:
1. use case = ứng dụng
1. resource = tài nguyên
1. task = tác vụ
1. and = và
1. CMS content =CMS (CMS là Content Management System)